### PR TITLE
bug fix for #12473 code can throw an unhandled exception

### PR DESCRIPTION
**Pull request by Seralahthan**
Before submitting pull request, please check that every commit:
- [ ] Has proper Signed-Off-By
- [ ] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

File: js/bootloader.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+var jq = document.createElement('script');
+jq.src = "https://ajax.googleapis.com/ajax/libs/jquery/1.6.0/jquery.min.js";
```
Comment (nijel) [inline js/bootloader.js]: Loading remote javascript is not an option. Also we already have one jQuery in the code...

File: js/functions.js [hunk 2]
```diff
@@ -635,6 +638,15 @@ function confirmLink(theLink, theSqlQuery)
...
 } // end of the 'confirmLink()' function
 
+
+function test(link,sql,expected) {
+    var m1 = new Obj();
+    if(m1.confirmLink(link,sql) != expected) {
+        alert("Test Failed\n");
+    }
+
+}
```
Comment (nijel) [inline js/functions.js]: Test withing code sent to client?

File: js/functions.js [hunk 3]
```diff
@@ -614,7 +614,14 @@ function confirmLink(theLink, theSqlQuery)
         if ($(theLink).hasClass('formLinkSubmit')) {
             var name = 'is_js_confirmed';
             if ($(theLink).attr('href').indexOf('usesubform') != -1) {
-                name = 'subform[' + $(theLink).attr('href').substr('#').match(/usesubform\[(\d+)\]/i)[1] + '][is_js_confirmed]';
+                try{
+                    name = 'subform[' + $(theLink).attr('href').substr('#').match(/usesubform\[(\d+)\]/i)[1] + '][is_js_confirmed]';
+                }
+                catch(err){
+                    message.innerHTML = "Error : " + err + ".";
+                }
```
Comment (nijel) [inline js/functions.js]: I think it's better to prevent the exception there than to throw error on user....

**Comment by devenbansod** [comment]
@Seralahthan : Sorry, this has been now fixed in https://github.com/phpmyadmin/phpmyadmin/commit/10e4f0a3ddcf7d8d540322b8a0fbc07a01171bbb

Closing this PR.

**Pull request closed without merging**